### PR TITLE
Frontend updates

### PR DESCRIPTION
**Pull request by totop716**
- Fix typo in landing page
- Move messages in landing page to JSON for localization
- Update github release workflow

### CODE REVIEW

File: .github/workflows/dev_release.yaml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Bump version and push tag
+        id: github_tag_action
+        uses: mathieudutour/github-tag-action@v6.0
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          release_branches: master
```
Comment (FriskyHamTitz) [inline .github/workflows/dev_release.yaml:25]: Shouldn't this point to dev not master?

**Review by FriskyHamTitz — CHANGES_REQUESTED** [review]

File: .github/workflows/dev_release.yaml (hunk 1 above)
Comment (totop716) [reply .github/workflows/dev_release.yaml]: Just updated the branch to dev

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -83,4 +84,31 @@ jobs:
         env:
           # Zone is required by both authentication methods
           CLOUDFLARE_ZONE: ${{ secrets.CLOUDFLARE_ZONE_ID }}
-          CLOUDFLARE_TOKEN: ${{ secrets.CLOUDFLARE_TOKEN }}
\ No newline at end of file
+          CLOUDFLARE_TOKEN: ${{ secrets.CLOUDFLARE_TOKEN }}
+
+      - name: Create GitHub Release
+        id: create_release
```
Comment (FriskyHamTitz) [inline .github/workflows/release.yaml:90]: Where did this new code come from? Why do we need it?

File: src/constants/stake.json [hunk 3]
```diff
@@ -1,5 +1,27 @@
 {
   "syrup": [
+    {
+      "token": "TOWER",
```
Comment (FriskyHamTitz) [inline src/constants/stake.json:4]: Was this included on masters, if so remove the code and then we'll make a seperate PR to rebase dev with master.

File: src/utils/useUSDCPrice.ts [hunk 4]
```diff
@@ -20,7 +20,12 @@ import { useDQUICKtoQUICK } from 'state/stake/hooks';
  */
 export default function useUSDCPrice(currency?: Currency): Price | undefined {
   const { chainId } = useActiveWeb3React();
-  const wrapped = wrappedCurrency(currency, chainId);
+  let wrapped = wrappedCurrency(currency, chainId);
```
Comment (FriskyHamTitz) [inline src/utils/useUSDCPrice.ts:23]: Rebase dev from master and remove code

**Review by FriskyHamTitz — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yaml (hunk 2 above)
Comment (totop716) [reply .github/workflows/release.yaml:90]: This is same as `release.yaml` in master branch

File: src/constants/stake.json (hunk 3 above)
Comment (totop716) [reply src/constants/stake.json]: Removed these updates

File: src/utils/useUSDCPrice.ts (hunk 4 above)
Comment (totop716) [reply src/utils/useUSDCPrice.ts]: Remove these updates

**Review by FriskyHamTitz — APPROVED** [review]

**Pull request merged**